pydy/tests/test_joints.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+SYMPY_VERSION = sm.__version__
+
+from ..bodies import Body
+from ..joints import PinJoint
+from ..system import System
+
+
+class TestPinJoint():
+    def setup(self):
+        self.sys = System(method='joints')
+
+    def test_joints_parent_child_relationship(self):
+        body = Body('body')
+        pin_joint = PinJoint(self.ground, 0, body, sys.reference_frame.x * Symbol('l'))
```

Comment:
User can pass any reference_frame provided there is a relation between the reference frame and the body.